.github/workflows/linux-build.yml

```diff
@@ -0,0 +1,365 @@
+name: linux-build
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu18-minimal:
+    name: ubuntu 18.04 minimal
+    runs-on: [ubuntu-18.04]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: |
+        sudo apt-get update
+        sudo apt-get install gfortran \
+        libopenmpi-dev openmpi-common openmpi-bin \
+        libblas-dev liblapack-dev libblas3 liblapack3
+
+        # Install cmake via candi to hit the version 3.16.3
+
+    - name: info
+      run: |
+        g++ -v
+        mpic++ -v
+        cmake --version
+    - name: build
+      run: |
+        # Modify some variables of the configuration script
+
+        # Install only DEBUG mode to save some time and disk space
+        echo 'DEAL_CONFOPTS="-D CMAKE_BUILD_TYPE=Debug"' >> candi.cfg
+
+        # Remove build directory after successful installation
+        echo 'INSTANT_CLEAN_BUILD_AFTER_INSTALL=ON' >> candi.cfg
+
+        # Remove downloaded packed src after successful installation
+        echo 'INSTANT_CLEAN_SRC_AFTER_INSTALL=ON' >> candi.cfg
+
+        # Remove unpack directory after successful installation
+        echo 'INSTANT_CLEAN_UNPACK_AFTER_INSTALL=ON' >> candi.cfg
+
+        # Skip examples build
+        echo 'BUILD_EXAMPLES=OFF' >> candi.cfg
+
+        # dealii version
+        echo 'DEAL_II_VERSION=v9.3.0-rc1' >> candi.cfg
```

Comment:
This is much better to be deleted, as we will automatically test the current stable version then.

Reply:
This was a relict of the draft, sorry. At this point I would suggest to switch the dealii version to master on the candi-master branch, as it looks like each dealii-version has its own candi-branch?!